entities/weapons/door_ram/shared.lua

```diff
@@ -85,14 +85,18 @@ local function ramDoor(ply, trace, ent)
     end
 
     -- Be able to open the door if any member of the door group is warranted
-    if GAMEMODE.Config.doorwarrants and ent:getKeysDoorGroup() and RPExtraTeamDoors[ent:getKeysDoorGroup()] then
-        allowed = false
-        for k,v in pairs(player.GetAll()) do
-            if table.HasValue(RPExtraTeamDoors[ent:getKeysDoorGroup()], v:Team()) and canRam(v) then
-                allowed = true
-                break
-            end
-        end
+    local keysDoorGroup = ent:getKeysDoorGroup()
+    if GAMEMODE.Config.doorwarrants and keysDoorGroup then
+        local teamDoors = RPExtraTeamDoors[keysDoorGroup]
+        if teamDoors then
+			allowed = false
+			for _, v in ipairs(player.GetAll()) do
+				if table.HasValue(teamDoors, v:Team()) and canRam(v) then
+					allowed = true
+					break
+				end
+			end
+		end
```

Comment:
Spacing appears to be tabs here (lines 92-99).

Reply:
Should be corrected.